### PR TITLE
feat: run migration as seperate job

### PR DESCRIPTION
**Pull request by cl3m0**
migrations should not run as part of pod deployments but only once as a seperate job. 

- seperated out LicensingServiceAccount
- extended makefile

### CODE REVIEW

**Review by rschach1 — APPROVED** [review]
OK, ich brauch da morgen nochmal ein paar Erklärungen. Aber lass uns mal läänden

File: k8s/lib/charts/migration-job.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      image,
+      nodeSelector,
+      migrationJobResources,
+      serviceAccountName,
+      postgresSecret,
+      segment,
+      name,
+    } = props;
+
+    // Ideally we should not create same config map again
```
Comment (saurabh2590) [inline k8s/lib/charts/migration-job.ts:65]: Change request: this comment is misleading and wrong copy paste.

**Review by saurabh2590 — CHANGES_REQUESTED** [review]
A tiny offending comment that threw me off.

**Review by saurabh2590 — APPROVED** [review]
Looks good to me.

**Pull request merged**